packages/bpk-component-navigation-stack/package.json

```diff
@@ -22,7 +22,7 @@
   "devDependencies": {
     "bpk-component-button": "^3.2.13",
     "bpk-component-icon": "^5.0.35",
-    "bpk-component-navigation-bar": "^2.1.31",
+    "bpk-component-navigation-bar": "^2.1.32",
```

Comment:
You won't need to update this as this will be done during the release process 😄

Reply:
Reverted!

Reply:
And re-reverted